config/config.go

```diff
@@ -48,6 +48,8 @@ type Config struct {
 	Version               string              `json:"-" yaml:"-" mapstructure:"-"`
 	APIServerURL          string              `json:"-" yaml:"-" mapstructure:"-"`
 	GitHubClientID        string              `json:"-" yaml:"-" mapstructure:"-"`
```

Comment:
For RegistryId, can you please use RegistryIds and array instead of string, it will be more useful